Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM alpine:3.11.3 as base
-RUN apk add git python3 gcc g++ linux-headers make
+FROM alpine:3.14.0 as base
```

Comment:
v3.14.0 is the latest version at the moment. No particular reason for updating it, other than future proving it. I tested that with works with Node v12 and another version (can't remember if the other version was v14 or v16 or v17).

Reply:
Sounds good